setup.py

```diff
@@ -20,7 +20,7 @@ def read(fname):
 
 setup(
     name='terrafort',
-    version='0.2.1',
+    version='0.2.3',
```

Comment:
why not `0.2.2`?

Reply:
I messed up the last release, forgot to update this for 0.2.2